xbmc/FileItem.cpp

```diff
@@ -2021,7 +2021,7 @@ void CFileItemList::Sort(SortDescription sortDescription)
   }
 
   // replace the current list with the re-ordered one
-  m_items.assign(sortedFileItems.begin(), sortedFileItems.end());
+  m_items = std::move(sortedFileItems);
```

Comment:
this likely won't do anything as we haven't implemented a move operator in CFileItem or CFileItemList, we probably should do that though